monetio/models/raqms.py

```diff
@@ -9,14 +9,16 @@
 import xarray as xr
 
 
-def open_dataset(fname, *, surf_only=False):
+def open_dataset(fname, convert_to_ppb=True, surf_only=False, **kwargs):
```

Comment:
Currently `kwargs` are unused. Did you want to pass them to something?
```suggestion
def open_dataset(fname, convert_to_ppb=True, surf_only=False):
```